tasks/IWantThis.tex

```diff
@@ -0,0 +1,91 @@
+\section{I Want This [Party Host]}
+A guest at the party speaks English, but with only a limited vocabulary. They want a drink, an hors d'oeuvre, and a set of utensils, but do not know the words to describe them. As such, they will ask the robot for one through gesturing, and will ask for recommendations made by the robot through gestures. They will also discuss a picture on the wall, which the robot will determine by analyzing their gaze.
+
+\subsection{Focus}
+Joint attention is a well-studied and important task in Human-Robot Interaction. The goal of this task is to really challenge the teams to perform a hard HRI task.
+
+\subsection{Main Goal}
+The robot must interpret 3 point gestures of varying difficulty, must produce the same 3 point gestures, and must interpret the human's gaze gesture as well as generate one, and interpret the establishment of mutual gaze.
```

Comment:
Looks too complex to me. I would split as explained below and request tops one with the other two as bonus objectives

Reply:
What exactly is meant by 3 point gestures? Are the gestures different or just pointing a different things?